### DISCUSSION TITLE
V4 Chips Component was Deprecated, AutoComplete with the  multiple param does not allow the very convenient addOnBlur or separator params

### DISCUSSION THREAD

**Discussion by duganbrianj** [primefaces/community#2889] [PrimeVue]
AutoComplete with the multiple param does not currently have two very convenient features of the Chips component. The chips component worked much better for allowing users to add their own inputs because of the addOnBlur and separator params.

Would be awesome to have that functionality added to the AutoComplete component. Until then, my preference would be to continue using the Chips component. How long would it be until the Chips component is fully removed?

**Comment by cagataycivici** [comment]
No immediate plans to remove the deprecated components, please create an issue ticket or send a PR to add these props to the AutoComplete.
